app/views/layouts/overdue.html.erb

```diff
@@ -0,0 +1,99 @@
+<% content_for(:content) do %>
+  <!---------------------
+  MOBILE NAV
+  --------------------->
+  <div class="mobile">
+    <div class="mt-0 mb-4 active-filters-mobile">
+      <a class="filters-modal-toggle" data-toggle="modal" data-target="#filter-modal">
+          <span class="filter-tags">
+            <% (Array(@patient_search_filters)).each do |filter, label| %>
+              <span class="badge badge-pill badge-secondary"><%= label %></span>
+            <% end %>
+          </span>
+        <i class="fas fa-filter"></i> Search filters
+      </a>
+    </div>
+    <!-- Filter Modal -->
+    <div class="modal fade" id="filter-modal" tabindex="-1" role="dialog" aria-labelledby="filter-modal" aria-hidden="true">
+      <div class="modal-dialog modal-dialog-centered" role="document">
+
+        <div class="modal-content">
+          <div class="modal-header">
+            <h2 class="modal-title" id="exampleModalCenterTitle">Patients</h2>
+            <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+              <span aria-hidden="true">&times;</span>
+            </button>
+          </div>
+          <div class="modal-body">
+            <div class="row">
+              <div class="col-6 mb-3">
+                <h4>Patients</h4>
+                <%= form_tag appointments_path, method: :get do %>
+                  <% @search_filters.each do |filter, label| %>
+                    <div class="checkbox-row">
+                      <label for=<%= "search-#{filter}-mobile" %>>
+                        <% checked = @search_filters.include?(filter) %>
+                        <%= check_box_tag("search_filters[]", filter, checked, { id: "search-#{filter}-mobile" } ) %>
+                        <%= label %>
+                      </label>
+                    </div>
+                  <% end %>
+                <% end %>
+              </div>
+            </div>
+          </div>
+          <div class="modal-footer">
+            <a href="?" type="button" class="btn btn-secondary">Clear all</a>
+            <button type="submit" form="query-filters-mobile" class="btn btn-primary">DONE</button>
+          </div>
+
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <!---------------------
+  DESKTOP NAV
+  --------------------->
+  <div class="row">
+    <div class="col-lg-2 pl-lg-0 desktop">
+      <section class="secondary-nav-filters">
+        <h4>Patients</h4>
+
+        <%= form_tag appointments_path, method: :get do %>
+          <%= hidden_field_tag :facility_id, @facility_id %>
+          <%= hidden_field_tag :per_page, @per_page %>
```

Comment:
We needed to do this because of our two forms and how they are using GET requests.  See https://github.com/simpledotorg/simple-server/pull/907/commits/0275fc5be68b7a0d10e5725b3b591f9a0fe1031e for details.

Reply:
After thinking about this a bit, I'd really rather do this without javascript, which would require the layout changes we discussed while pairing.

This really feels to me like it should be one form that lets you filter on facilities, search filters, or whatever else we come up with. Adding JS for a pretty straightforward 'filter' endpoint would add a lot of unnecessary complexity.

Reply:
Could setting these inputs as `<input form="sidebar-form-id">` solve this? It allows you to have an input field belonging to a form outside the form. 

This is being done in My Facilities > BP control period selection, for example: There are "two forms with shared state" but effectively they're the same form thanks to the `form` attribute.

Reply:
This is really cool, is it a vanilla HTML feature?

Reply:
Yes! 😄

Reply:
> Could setting the `facility` and `per_page` filters as `<select form="sidebar-form-id">` solve this? It allows you to have an input field belonging to a form outside the form.

Cool, I'm doing this with everything going through the form in the sidebar when on the overdue page. Thanks for the suggestion!